src/main/main.cpp

```diff
@@ -175,6 +176,7 @@ int main(int argc, char *argv[])
 #elif defined(Q_OS_LINUX)
     bool isLinux = true;
     bool isTails = TailsOS::detect();
+    bool isWhonix = WhonixOS::detect();
```

Comment:
note how isTails is defined as a bool on line 157 to stop none linux OS throwing the not defined error, same is needed for isWhonix